.ci/azure-pipelines.yml

```diff
@@ -10,38 +24,42 @@ jobs:
         persistCredentials: true
 
       - task: CmdLine@2
-        displayName: "Clone Server"
+        displayName: "Clone Server Branch"
+        condition: in(variables['Build.Reason'], 'IndividualCI')
+        inputs:
+          script: "git clone --single-branch --branch $(Build.SourceBranchName) --depth=1 https://github.com/jellyfin/jellyfin.git $(Build.SourcesDirectory)/jellyfin"
+
+      - task: CmdLine@2
+        displayName: "Clone Server Target"
+        condition: in(variables['Build.Reason'], 'PullRequest')
         inputs:
-          script: "git clone --single-branch --branch master --depth=1 https://github.com/jellyfin/jellyfin.git $(Build.SourcesDirectory)/jellyfin"
+          script: "git clone --single-branch --branch $(System.PullRequest.TargetBranch) --depth=1 https://github.com/jellyfin/jellyfin.git $(Build.SourcesDirectory)/jellyfin"
```

Comment:
This was intentionally not done, since the branch might not exist (feature and release branch for example). We may want to add a secondary clone that runs if this one fails (and clones master).

Reply:
I never mentioned why I used master for this step :laughing: and it was because I didn't want to bother at the time, not the reason you mentioned. I agree with your suggestion though.